src/index.html

```diff
@@ -1,17 +1,155 @@
+            id="age"
+            min="0"
+            max="100"
+            value="12"
+            >
+          </label>
+          <label for="birthDate" class="control-label">
+            Full date of birth:
+            <input
+            type="date"
+            name="birthDate"
+            id="birthDate"
+            >
+          </label>
+          <label for="acceptAgreement" class="control-label">
+            I accept the term of the agreement
+            <input
+            type="checkbox"
+            name="acceptAgreement"
+            id="acceptAgreement"
+            value="no"
+            required
+            >
+          </label>
+        </fieldset>
+        <fieldset class="fieldset">
+          <legend>Registration</legend>
+          <label for="email" class="control-label">
+            E-mail:
+            <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+            >
+          </label>
+          <label for="password" class="control-label">
+            Password:
+            <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="6"
+            maxlength="20"
+            required
+            >
+          </label>
+        </fieldset>
+        <fieldset class="fieldset">
+          <legend>An interesting fact about you!</legend>
+          <label class="control-label">
+            Do you love cats?
+            <input
+            type="radio"
+            name="cats"
+            value="yes"
+            > Yes 
+            <input
+            type="radio"
+            name="cats"
+            value="no"
+            > No 
+          </label>
+          <label class="control-label">
+            What is your favorite color?
+            <input type="color">
+          </label>
+          <label class="control-label">
+            What time do you go to bed?
+            <input
+            type="time"
+            step="1"
+            >
+          </label>
+          <label class="control-label"
+          >What are your favorite brand of cars?
+            <select name="favorite cars" multiple>
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+            </select>
+          </label>
+          <label class="control-label">
+            How do you rate our work?
+            <input
+            type="range"
+            min="0"
+            max="11"
+            value="0"
+            >
+          </label>
+        </fieldset>
+        <fieldset class="fieldset">
+          <legend>Additional info:</legend>
+          <label class="control-label">
+            Comments:
+            <textarea
+            name="comments"
+            cols="20"
+            rows="2"
+            autocomplete="off"
+            >
+            </textarea>
```

Comment:
```suggestion
            <textarea
            name="comments"
            cols="20"
            rows="2"
            autocomplete="off"
            ></textarea>
```
<img width="882" alt="image" src="https://user-images.githubusercontent.com/102411130/216550479-f7b82050-cd82-40b3-b1a3-e129c52037e5.png">
 check this comment. now your `textarea` has space

Reply:
Я не можу откриваючий  тег `textarea`(останній знак більше )  поставити в один рядок із закриваючим, тоді при npm run lint - видає помилку, що вони повинні бути на одному рівні